Google.Api.Generator/CodeGenerator.cs

```diff
@@ -84,19 +85,21 @@ internal static class CodeGenerator
         };
 
         public static IEnumerable<ResultFile> Generate(FileDescriptorSet descriptorSet, string package, IClock clock,
-            string grpcServiceConfigPath, IEnumerable<string> commonResourcesConfigPaths)
+            string grpcServiceConfigPath, string serviceConfigYamlPath, IEnumerable<string> commonResourcesConfigPaths)
```

Comment:
just 'serviceConfigPath' for consistency with tests?

Reply:
*Could* do, although given that we've had the gRPC service config this whole time, it feels worth clarifying which one we're really talking about.

Reply:
Done in the end.